change/@fluentui-react-native-tester-win32-cf0ffbe6-2dd9-403a-bebc-c0c274c66e61.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Port partner focuszone tests to FURN",
```

Comment:
Seems you have a couple of change files from other PRs here (for the menu e2e test rewrite and additional focuszone tests). I would delete them.